account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -233,6 +233,7 @@ def _recompute_suspense_line(self, data, reconcile_auxiliary_id, manual_referenc
         new_data = []
         suspense_line = False
         counterparts = []
+        suspense_currency = self.foreign_currency_id or self.currency_id
```

Comment:
The suspense currency should always be the foreign_currency if set (at least it is the case on enterprise)